apps/condo/domains/organization/access/OrganizationEmployee.js

```diff
@@ -48,8 +49,8 @@ async function canManageOrganizationEmployees ({ authentication: { item: user },
             id: employeeForUser.role,
             organization: { id: employeeForUser.organization },
         })
-
-        return !!(employeeRole && employeeRole.canManageEmployees)
+        const createPermission = await checkPermissionInUserOrganizationOrRelatedOrganization(user.id, organizationId, 'canCreateEmployees')
```

Comment:
A name `createPermission` looks like an action

```suggestion
        const permissionOnCreate = await checkPermissionInUserOrganizationOrRelatedOrganization(user.id, organizationId, 'canCreateEmployees')
```